app/assets/javascripts/discourse/app/lib/to-markdown.js

```diff
@@ -315,6 +315,18 @@ export class Tag {
           return text;
         }
 
+        if ("hashtag-cooked" === attr.class) {
```

Comment:
It doesn’t seem very future proof to check for exact class value instead of checking if a specific class is present.

Reply:
True, will address in another PR